srcpkgs/wezterm/template

```diff
@@ -0,0 +1,76 @@
+# Template file for 'wezterm'
+pkgname=wezterm
+version=20220624
+revision=1
+_srcver=20220624-141144-bd1b7c5d
+wrksrc="${pkgname}-${_srcver}"
+build_style=cargo
+build_helper="rust"
+make_check_args="--workspace --locked --exclude wezterm-ssh"
+hostmakedepends="cargo pkg-config python3"
+makedepends="nautilus-python fontconfig-devel freetype-devel harfbuzz-devel
+ libX11-devel libxkbcommon-devel libxkbcommon-x11 openssl-devel python3-devel
+ rust-std wayland-devel xcb-util-devel xcb-util-image-devel
+ xcb-util-keysyms-devel xcb-util-wm-devel"
+depends="appdata-tools desktop-file-utils"
+checkdepends="openssh"
+short_desc="GPU-accelerated cross-platform terminal emulator and multiplexer"
+maintainer="Joseph Benden <joe@benden.us>"
+license="MIT"
+homepage="https://wezfurlong.org/wezterm/"
+distfiles="https://github.com/wez/wezterm/releases/download/${_srcver}/${pkgname}-${_srcver}-src.tar.gz"
+checksum="898bfc12794f2e6b3bcd3d066963394ca537ef84a66278ec72e79e2937d99cbf"
+python_version=3
+
+if [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	# The platform has 4 bugs in processing emotes; this shouldn't effect everybody?!
+	#
+	# failures:
+	#   cell::test::teeny_string
+	#   cell::test::test_width
+	#   surface::line::test::hyperlinks
+	#   surface::test::double_width
+	#
+	make_check=no # i686 platform bugs
```

Comment:
maybe using `make_check_args+=" -- --skip ..."` could be used instead?

Reply:
Yeah!